install.sh

```diff
@@ -49,8 +56,8 @@ systemctl disable systemd-resolved.service
 systemctl stop systemd-resolved
 rm -rf /etc/resolv.conf
 echo "nameserver 1.1.1.1" > /etc/resolv.conf
-echo "options edns0" >> /etc/resolv.conf
-echo "search eu-north-1.compute.internal" >> /etc/resolv.conf
+echo "nameserver 1.0.0.1" > /etc/resolv.conf
```

Comment:
This replaces the previous row but I'll fix it later,

Reply:
Removed it, at is was AWS specific.